pallets/technical/src/lib.rs

```diff
@@ -275,9 +284,12 @@ pub mod pallet {
             + FromGenericPair
             + MaybeSerializeDeserialize
             + common::ToFeeAccount
-            + common::ToTechUnitFromDEXAndTradingPair<
+            + common::ToXykTechUnitFromDEXAndTradingPair<
```

Comment:
Do we really need this trait bounds here?

Reply:
Discussed in DMs, haven't found a good way to move the bounds to dependant pallets